### DISCUSSION TITLE
ElasticSearch 7  Method not found org.elasticsearch.client.Request.addParameters

### DISCUSSION THREAD

**Discussion by shekarpcs** [Netflix/conductor-community#130] [Q&A]
I receive following error when I set 
conductor.elasticsearch.version=7

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
8641 [main] ERROR org.springframework.boot.diagnostics.LoggingFailureAnalysisReporter [] - 

***************************
APPLICATION FAILED TO START
***************************

Description:

An attempt was made to call a method that does not exist. The attempt was made from the following location:

    com.netflix.conductor.es7.dao.index.ElasticSearchRestDAOV7.waitForHealthyCluster(ElasticSearchRestDAOV7.java:324)

The following method did not exist:

    'void org.elasticsearch.client.Request.addParameters(java.util.Map)'

The method's class, org.elasticsearch.client.Request, is available from the following locations:

    jar:file:/Users//conductor-config/conductor-server-3.5.1-boot.jar!/BOOT-INF/lib/elasticsearch-rest-client-6.8.12.jar!/org/elasticsearch/client/Request.class

The class hierarchy was loaded from the following locations:

    org.elasticsearch.client.Request: jar:file:/Users//conductor-config/conductor-server-3.5.1-boot.jar!/BOOT-INF/lib/elasticsearch-rest-client-6.8.12.jar!/

**Comment by apanicker-nflx** [comment]
Please follow all the steps listed [here](https://github.com/Netflix/conductor/blob/main/es7-persistence/README.md) to enable using  ES7.

**Comment by manan164** [comment]
Hi @shekarpcs , The reason for the conflicting dependency is to generate a lock file again.  Please delete all the `dependencies.lock ` file from all modules and run command, `./gradlew generateLock updateLock saveLock`
